docs/guides/configuration.md

```diff
@@ -52,21 +47,39 @@ Puppeteer will look up the file tree for any of the following formats:
 See the [`Configuration`](../api/puppeteer.configuration) interface for possible
 options.
 
-:::caution
+### Changing download options
 
-After adding a configuration file, you may need to remove and reinstall
-`puppeteer` for it to take effect if the changes affect installation.
+When the changes to the configuration include changes to download option,
+you will need to re-run postinstall scripts for them to take effect.
 
-:::
+This can most easily be done with running
 
-:::caution
+```bash npm2yarn
+npm rebuild puppeteer
```

Comment:
let's encourage the use of `npx puppeteer browsers install` which reminds me, let's change the CLI to allow installing all configured browsers if install is run w/o args